src/sentry/data_export/endpoints/data_export.py

```diff
@@ -47,6 +48,12 @@ def post(self, request, organization):
             return Response(serializer.errors, status=400)
         data = serializer.validated_data
 
+        if len(data["query_info"].get("field", [])) > MAX_FIELDS:
+            detail = "You can export up to {0} fields at a time. Please delete some and try again.".format(
+                MAX_FIELDS
+            )
+            return Response({"detail": detail}, status=400)
```

Comment:
You could use `rest_framework.exceptions.ParseError` for these.

Reply:
Thanks for the tip! I updated it.